server/form-pages/apply/risks-and-needs/risk-to-self/currentRisk.ts

```diff
@@ -23,6 +24,12 @@ export default class CurrentRisk implements TaskListPage {
 
   body: CurrentRiskBody
 
+  importDate = this.application.data?.['risk-to-self']?.['current-risk'].dateOfOasysImport
+    ? DateFormats.isoDateToUIDate(this.application.data?.['risk-to-self']?.['current-risk'].dateOfOasysImport, {
```

Comment:
I *think* that the page name could also be null/undefined if no Oasys was imported?.. so would have to be `this.application.data?.['risk-to-self']?.['current-risk']?.dateOfOasysImport` ?

Reply:
good thinking 💡